backend/src/org/commcare/resources/model/installers/CacheInstaller.java

```diff
@@ -46,24 +47,20 @@
         return cacheStorage;
     }
 
-    int cacheLocation;
+    @Override
```

Comment:
Hmm, yeah, it isn't useful here since the parent implementation is also abstract. We can take it later